app/_src/gateway/support/index.md

```diff
@@ -21,7 +21,13 @@ For the purposes of this support document:
 
 Kong introduces major functionality and breaking changes by releasing a new major version. Major version releases happen rarely and are usually in response to changes in major industry trends, significant architectural changes or significant internal product innovation. There is no regular release cadence of major versions.
 
-Kong aims to release a new minor version every 10 weeks. Minor versions contain features and bug fixes. Minor versions are backwards compatible within that major version sequence.  Every minor version is supported for a period of 1 year from date of release. This is done by releasing patches that apply to each supported minor version. Customers are encouraged to keep their installations up to date by applying the patches appropriate to their installed version. All patches released by Kong are roll-up patches (for example, patch 1.5 for release version 3.3 includes all the fixes in patches 1.4, 1.3, 1.2, and 1.1).  
+Kong aims to release a new minor version every 10 weeks. Minor versions contain features and bug fixes. Minor versions are backwards compatible within that major version sequence. Every minor version is supported for a period of 1 year from date of release. This is done by releasing patches that apply to each supported minor version. Customers are encouraged to keep their installations up to date by applying the patches appropriate to their installed version. All patches released by Kong are roll-up patches (for example, patch 1.5 for release version 3.3 includes all the fixes in patches 1.4, 1.3, 1.2, and 1.1).
+
+{:.important}
+> **Important:** There can be exceptions to the semantic versioning model. 
+Due to backports, new features and breaking changes are possible at any version level, including patch versions.
+To avoid issues, do not upgrade to any new version automatically, and 
+make sure to review all relevant [changelog entries](/gateway/changelog/) before manually upgrading your deployments.
```

Comment:
To clarify: I'm fairly certain that any of this can happen in any patch version, LTS or not. Is this true? Or do I need to adjust this note to say something like "Because of its long-lived nature, breaking changes can happen in patches for an LTS release"? Either way, new features can happen in any version.